File: lessons/compiled_lang/cs/src/k00.parameters.cs
using System;

class K00Parameters
{
    static void Celsius(double fahrenheit)
    {
        double temperature = (fahrenheit - 32.0) * 5 / 9;
        Console.WriteLine("The Celsius temperature is " + temperature.ToString("F1") + " degrees.");
    }

    static void Main()
    {
        double temperature = 73;
        Celsius(temperature);
    }
}

File: lessons/compiled_lang/cpp/src/k00.parameters.cpp
#include <cstdio>

void celsius(double fahrenheit) {
    double temperature = (fahrenheit - 32.0) * 5 / 9;
    printf("The Celsius temperature is %.1f degrees.\n", temperature);
}

int main() {
    double temperature = 73;
    celsius(temperature);

    return 0;
}
